apps/tlon-mobile/src/hooks/useFocusNotifyOnChangeProps.ts

```diff
@@ -0,0 +1,31 @@
+import { useFocusEffect } from '@react-navigation/native';
+import { NotifyOnChangeProps } from '@tanstack/query-core';
+import React from 'react';
+
+export function useFocusNotifyOnChangeProps(
+  notifyOnChangeProps?: NotifyOnChangeProps
+): NotifyOnChangeProps {
+  const focusedRef = React.useRef(true);
+
+  useFocusEffect(
+    React.useCallback(() => {
+      focusedRef.current = true;
+
+      return () => {
+        focusedRef.current = false;
+      };
+    }, [])
+  );
+
+  return () => {
```

Comment:
These default return values are confusing to me. Some comments about what's happening there or more broadly, how this hook is supposed to be used would be helpful